libs/blocks/global-navigation/global-navigation.css

```diff
@@ -571,6 +570,11 @@ header.global-navigation {
     overflow: auto;
   }
 
+  .global-navigation.has-promo .feds-navItem--megaMenu .feds-popup {
+    max-height: calc(100vh - 100% - var(--global-height-navPromo));
+    box-sizing: border-box;
```

Comment:
Most of the elements in the global navigation are `context-box` and this might generate some confusion when setting padding

Reply:
@narcis-radu Yes, but this case involves the megamenu, where scrolling is necessary if its height and width exceed the screen dimensions. To address this, we calculate the height for the megamenu and add a scroll. However, due to `content-box`, it only sets the scroll for the content area, but we need scrolling for both the content and the surrounding spacing. I've now moved it closer to `overflow:auto`, which is only required for the scroll functionality.

Reply:
We already have scrolling functionality on Production and the box-sizing is `content-box`, don't we ?

Reply:
I think this was needed to compensate the 40px padding which was not included in height and the bottom-most link in the large menu was getting cut off from the scrolling.
![image](https://github.com/user-attachments/assets/cc87f525-d280-4cdc-8a4b-b2a792739d7d)